<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newReservation</class>
 <widget class="QDialog" name="newReservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nowa rezerwacja</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Film</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="movieCB"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Data oraz godzina</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="dateCB"/>
   </item>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Imie</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLE"/>
   </item>
   <item>
    <widget class="QLabel" name="surnameLabel">
     <property name="text">
      <string>Nazwisko</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="surnameLE"/>
   </item>
   <item>
    <widget class="QLabel" name="ticketLabel">
     <property name="text">
      <string>Ilość biletów</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ticketLE">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="infoBrowser">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Wybierz miejsca</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
